scss/_patterns_grid.scss

```
@@ -37,7 +37,7 @@
 @mixin vf-p-grid {
   // FIXME: this should be removed from framework SCSS
   // (see https://github.com/canonical/vanilla-framework/issues/3199)
-  .grid-demo [class*='#{$grid-column-prefix}'] {
+  .grid-demo [class*='col'] {
```

This would match ALL class names containing "col" string, this may lead to very unexpected results.

While it's a similar case with `col-` it at least is a bit safer as it has the structure of col- column, but for a name like `.col` we should explicitly only match that class name.


```suggestion
  .grid-demo .col,
  .grid-demo [class*='#{$grid-column-prefix}'] {
```